src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -930,8 +932,19 @@ void primitive_inst::do_runtime_skip_gather() {
     auto idx_shape = _impl_params->get_input_layout(1).get_shape();
```

Comment:
Please add condition in line 925 to return if get_node().can_be_optimized() is false.

Reply:
Done.